app/models/card.rb

```diff
@@ -63,7 +74,7 @@ def processing_failure_translation_update
   private
     def original_text_cannot_be_equal_translated_text
       if original_text.mb_chars.downcase == translated_text.mb_chars.downcase
-        errors.add(:translated_text, 'Переведенный текст не должен совпадать с оригиналом!')
+        errors.add(:translated_text, I18n.t('views.cards.flash_messages.original_text_not_equal_translated_text'))
```

Comment:
Line is too long. [114/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.